pages/linux/wami.md

```diff
@@ -0,0 +1,20 @@
+# wami
+
+> A tool that recommends suitable programs for specific tasks. Open-source, easy to use.
```

Comment:
Maybe we can reword this line to stay consistent with other pages using "open source" and "easy to use" terms.

```suggestion
> An open-source and easy-to-use tool that recommends suitable programs for specific tasks.
```

Reply:
Thank you